glean-core/ios/Glean/Glean.swift

```diff
@@ -105,7 +111,47 @@ public class Glean {
 
     /// Handle background event and send appropriate pings
     func handleBackgroundEvent() {
-        // sendPings()
+        // Perform task on a background thread
+        _ = Dispatchers.shared.launch {
+            Glean.backgroundTaskID = UIApplication.shared.beginBackgroundTask(withName: "Glean Upload Task") {
+                // End the task if time expires
+                UIApplication.shared.endBackgroundTask(Glean.backgroundTaskID)
+                Glean.backgroundTaskID = UIBackgroundTaskIdentifier.invalid
+            }
+
+            self.sendPingsByName(pingNames: ["baseline", "events"])
```

Comment:
Why are you adding `events`? We don't have any event metric type. May be clearer to have this later on, once the rest is added

Reply:
Fun fact: if this stays the _only_ thing we need to enable events for users is the Swift wrapper.

All logic determining if events are there and need to be sent is in Rust already, so it will just start to work.

Therefore I'd be fine having it here already.

Reply:
Good point. Fine with me!

Reply:
I read this as "Glean, it just works!"